.github/workflows/ci.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Run Unit Tests
       run: |
         export PATH=${HOME}/Library/Python/3.8/bin:${HOME}/Library/Python/3.9/bin:${HOME}/.local/bin:${PATH}
-        meson -Dbuildtype=release -Denable_cli=true -Duse_gpl=false build-nogpl
+        meson -Dbuildtype=release -Denable_cli=true build-nogpl
```

Comment:
if `use_gpl` is not used anymore, please remove the key from `meson_options.txt` and from `meson.build`

Reply:
maybe remove this job too.

Reply:
it is not present in either of the files.

Reply:
https://github.com/rizinorg/rz-libdemangle/blob/main/meson_options.txt#L1
https://github.com/rizinorg/rz-libdemangle/blob/main/meson.build#L32